UserInterface/ViewModels/TestViewModel.cs

```diff
@@ -236,6 +237,7 @@ private void GetTest()
 
             ShowTestTargetAudienceView = "Hidden";
             Test = testService.GetTestByTargetAudienceIdAndActive(selectedTargetAudience.Id); // toegevoegd door jasper
+
             testProgressData = new TestProgressData(Test);
             testProgressData.CurrentQuestionNumber = 1;
```

Comment:
niks veranderd in deze file zou hem dan ook niet commiten.

Reply:
ohja